sdk/batch/batch-typespec/routes.tsp

```diff
@@ -2376,15 +2376,7 @@ then the directory must be empty or deletion will fail.
   @route("/pools/{poolId}/nodes/{nodeId}/files/{filePath}")
   @get
   GetFileFromBatchNode is RpcOperation<
-    BatchNodeFileClientPathParameters &
-      BatchModifiedSinceHeaders & {
-        @doc("""
-The byte range to be retrieved. The default is to retrieve the entire file. The
-format is bytes=startRange-endRange.
-""")
-        @header
-        `ocp-range`?: string;
-      },
+    GetFileFromBatchNodeOptions,
```

Comment:
A prototype of group these parameters to `GetFileFromBatchNodeOptions`

Enabled by dev-options.
```
    dev-options:
      support-quantum-model: true
```